packages/itwin/grouping-mapping-widget/src/widget/components/HorizontalTile.scss

```diff
@@ -10,8 +10,8 @@
   border-radius: 5px;
   background-color: var(--iui-color-background-2);
   height: $iui-baseline * 4;
-  padding: 5.5px $iui-s;
-  .body {
+  padding: 5.5px 8px;
```

Comment:
Why the style change in this PR?

Reply:
That was due to a merge issue, I must've forgotten overlooked it and forgot to change it back

Reply:
Done